src/sonic-host-services/scripts/check_readonly_disk

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+echo "checking for read-only disk"
+SYSTEM_ERR=4
+
+touch /var/log/syslog
```

Comment:
sometime, the /var/log is mounted as tmpfs ramdisk, so we need to check some other file in the file system, maybe we should touch /home/admin/.test_disk_readwrite file?

Reply:
ok

Reply:
/tmp can also be tmpfs which is not on the disk.